test/integration/tests/4453-detailed/Main.hs

```diff
@@ -0,0 +1,7 @@
+import StackTest
+import System.Directory
+
+main :: IO ()
+main = do
+    removeDirIgnore ".stack-work"
+    stack ["test"]
```

Comment:
We're using `defaultResolverArgs` where necessary, now, correct?

```suggestion
    stack [defaultResolverArgs, "test"]
```

Reply:
No idea, feel free to go ahead because I have no clue.

Reply:
@dbaynard `stack.yaml` is already using the current default, I'm not sure we need to override it with the same value

Reply:
I think it's for when we have to upgrade the resolver for new releases; this makes it clear that the integration test should work at whichever resolver is selected, rather than for a specific resolver.